liberty-ls/src/test/java/io/openliberty/tools/langserver/xml/LibertyConfigFileManagerTest.java

```diff
@@ -0,0 +1,56 @@
+package io.openliberty.tools.langserver.xml;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.eclipse.lsp4j.WorkspaceFolder;
+import org.junit.Test;
+
+import io.openliberty.tools.langserver.LibertyConfigFileManager;
+
+public class LibertyConfigFileManagerTest {
+    File resourcesDir = new File("src/test/resources");
+    public final String CUSTOM_SERVER_ENV_VALUE = "/user/sample-project/src/main/liberty/config/customserverenv.env";
+    public final String CUSTOM_BOOTSTRAP_PROPERTIES_VALUE = "/user/sample-project/src/main/liberty/config/custombootstrapprops.properties";
+    public final String WINDOWS_CUSTOM_SERVER_ENV_VALUE = "C:\\user\\sample-project\\src\\main\\liberty\\config\\customserverenv.env";
```

Comment:
I don't think hard coding the path with the drive letter is going to be ok.

Reply:
It should be okay for this test because it's a hard coded value in the test resource. Matt generated our test resource on Windows, which we slightly tweaked.